test/e2e/authorizer/authorizer_test.go

```diff
@@ -48,6 +48,17 @@ import (
 var embeddedResources embed.FS
 
 func TestAuthorizer(t *testing.T) {
+	if len(framework.TestConfig.Kubeconfig()) > 0 {
+		/* TODO: this test creates a clusterrole/clusterrolebinding in genericcontrolplane.LocalAdminCluster. Running
+		this test multiple times will not work in the current form because the names will collide. However, unique
+		names is not a sufficient solution, because any clusterrole/binding from a previous test run will pollute
+		the intent of the test. A possible fix would be to attempt to clean up any possible resources this test
+		intends to create in genericcontrolplane.LocalAdminCluster at the beginning of the test, or to split it
+		into parts that can run against a shared server, vs. those that truly cannot.
+		*/
+		t.Skip("Skipping until this test is updated to support multiple runs against a shared server")
+	}
+
```

Comment:
addressed in https://github.com/kcp-dev/kcp/pull/921

Reply:
Undone